gui/functionality/popup_window.py

```diff
@@ -1,20 +1,41 @@
+""" One of the helper widgets.
+
+    See: :class:`.ProgramInfoWindow`, :class:`.ShowLogsWindow`,
+    :class:`.YesOrNoWindow`, :class:`.ShowJobsWindow`,
+    :class:`.EditNativeArgumentsWindow`
+"""
 from enum import Enum
 from PyQt5.QtWidgets import QWidget, QMessageBox
 
 
 class WindowType(Enum):
+    """ Avialble popup window types.
+
+        :attr:`.success`: Window that shows a success message.
+
+        :attr:`.error`: Window that shows an error message.
+
+    """
     success = 1
     error = 2
 
 
 class PopUpWindow(QWidget):
+    """ Helper widget window for idact notebook native arguments.
```

Comment:
I think that this one is not only for native arguments

Reply:
yes, you are right, fixed